### PR TITLE
Initial commit of the environment.

### PR DESCRIPTION
**Pull request by HarmGriffioen**
- Added basic jUnit test
- Added jUnit to maven dependencies
- Set up maven
- Checkstyled

### CODE REVIEW

File: tygron-environment/pom.xml [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+	<artifactId>tygron_environment</artifactId>
+	<version>1.0-SNAPSHOT</version>
+	<packaging>jar</packaging>
+
+	<name>eis_pong</name>
+	<url>http://maven.apache.org</url>
+
+	<properties>
+		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+		<environmentname>pong</environmentname>
```
Comment (PauluzzNL) [inline tygron-environment/pom.xml]: Environment name is pong/ main class is PongInterface. Please rename appropriately.

**Comment by PauluzzNL** [comment]
Please check by editorial notes. If that's solved, I'll approve your PR. :smile:

**Comment by HarmGriffioen** [comment]
Updated with the feedback

**Comment by PauluzzNL** [comment]
Looking good now :+1:

**Comment by devbridie** [comment]
Let me pull it first and see if it's alright on my side.

**Comment by PauluzzNL** [comment]
Great update, looks good to me now :+1: :heavy_check_mark:

**Comment by devbridie** [comment]
My only comment was fixed as per 21674b8. :+1:

**Pull request merged**